src/lib/update-feeds.js

```diff
@@ -0,0 +1,79 @@
+import client from '../db/client';
+import getEvent from './events/get-event';
+import buildFeedItem from './events/build-feed-item';
+import getHostId from './events/get-host-id';
+import getInviteesIds from './events/get-invitees-ids';
+import saveFeedItem from './events/save-feed-item';
+import saveFeedItemForDeletedEvent from './events/save-feed-item-for-deleted-event';
+import PubSub from 'pubsub-js';
+
+const informCurrentUser = (method, url) => {
+  if (method === 'POST' || method === 'PUT') {
+    if ((/\/events\/\d+$/).test(url) || (/\/events$/).test(url)) {
+      return true;
+    }
+  }
+  return false;
+};
+
+export default function updateFeeds (req, res, next) {
+
+  const subject_user_id = req.subject_user_id;
+  const event_id = req.event_id;
+  const informAllInvitees = req.informAllInvitees;
+  const deletingEvent = req.method === 'DELETE' && (/\/events\/\d+/).test(req.url);
+
+  if (!subject_user_id || !event_id || informAllInvitees === undefined) {
```

Comment:
@heron2014 yes, because it's a boolean value, i don't want `informAllInvitees === false` to be misinterpreted